Solidity/README.md

```diff
@@ -40,12 +40,12 @@ cp ../.env.dist ../.env
     - **You cannot use the same private key for each chain.**
   - Chain ID
     - Depending on the network you're deploying to, you'll need to set the `CHAINID` variable to the correct chain ID.
-      - **Preset to 9000 for testnet deployments.**
-      - <u>Testnet</u>: `9000`
+      - **Preset to 9000 for testnet and devnet deployments.**
+      - <u>Testnet and Devnet</u>: `9000`
       - <u>Local</u>: `1337`
   - RPC Endpoints
-    - **RPC endpoints are default configured for existing Quai RPC endpoints.**
-    - Information for local and remote hosted RPC endpoints for each chain can be found [here](https://docs.qu.ai/build/networks).
+    - **The RPCURL variable defaults to the hosted Quai Network RPC endpoint.**
```

Comment:
Change "...hosted Quai Network RPC endpoint" to "...hosted Quai Network RPC URL"

Reply:
Changed in https://github.com/dominant-strategies/hardhat-example/pull/43/commits/b267aa2edc09cc1a8cd12ac344d03f1166316882